### ISSUE TITLE
Később jött hallgatók felvétele

### ISSUE THREAD

**Issue by Kisfejes** [bme-db-lab/szglab5-backend#51]
Később jött hallgatók felvétele a rendszerbe + szükséges egyedek generálása ( StudentRegistration, Event, Dliverables)

**Comment by csutorasr** [comment]
Szerintem ez mehetne az xls append alapjan.

A letezo azonositok (email vagy neptun) vizsgalata egyezesre. Ha letezik, akkor semmi, ha nem, akkor uj generalasa. Ha nincs semmi az adatbazisban, akkor mind uj generalasa lesz ertelem szeruen.

Kesobbi feature lehet, hogy a mar letrehozott felhasznaloknak is lehessen eventeket modositani, de ez nem fontos az inicializalashoz,

**Comment by csutorasr** [comment]
Erre most nem lesz szukseg elvileg, mert aki akarta mar felvette a targyat. @jmarton ha egyetertesz rakd at nice to have es v1.1-re. :)

**Comment by jmarton** [comment]
Egyetértek, @csutorasr. Egy-egy hallgató még persze előkerülhet, de az talán megy enélkül is.

**Comment by Kisfejes** [comment]
CLI tool készül a probléma megoldására #67
